### PR TITLE
docker: Docker Compose for developers

### PR DESCRIPTION
**Pull request by ghost**
To do this, must add new ELASTICSEARCH_HOST setting to app. Default localhost, as before.

### CODE REVIEW

**Comment by BibianaC** [comment]
@odscjames would it be good to add this in the `README`?

**Comment by odscjames** [comment]
@BibianaC Possibly, but I wasn't sure of some of the information structure so I thought let's just get it in and sort out structure later. Note `docs/developer/adding-facets.md` is already a docs file with no links to it, so this won't be the first such page.

**Review by BibianaC — APPROVED** [review]

File: docs/developer/docker-compose.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+(If you work for 360, the URL is `https://store.data.threesixtygiving.org/api/grantnav/updates` and the username and password can be found in [the private deploy repo](https://github.com/ThreeSixtyGiving/threesixtygiving-deploy-pillar-private).)
+
+First get the data. In your normal shell and Grantav Directory, run:
+
+```
+mkdir datastoredata
+cd datastoredata
+wget https://raw.githubusercontent.com/ThreeSixtyGiving/datastore/master/tools/poll_datastore.py
+python3 poll_datastore.py --url  XXXXXXXXXXX --username YYYYYYYYY --password ZZZZZZZZZZ --load-grantnav-script ls
+```
```
Comment (michaelwood) [inline docs/developer/docker-compose.md:40]: I wouldn't encourage people to use poll_datastore , really it's designed for use in a cron job for automatic updates.
```
wget <datastore url>/grantnanv_pacakges/ 
tar -xf package.tar.gz
import_to_elastic_search.py .data...
```

File: docs/developer/docker-compose.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+```
+$ docker-compose -f docker-compose.dev.yml run grantnav-web su -c 'wget -O  scvo-test-data.json https://scvo.scot/support/coronavirus/funding/scottish-government/community-recovery/crf/recipients/data.json && python dataload/import_to_elasticsearch.py --clean scvo-test-data.json'
+```
+
+## Loading Full Data
+
+To do this you will need to know the URL, username and password to a [DataStore server](https://github.com/ThreeSixtyGiving/datastore).
+
+(If you work for 360, the URL is `https://store.data.threesixtygiving.org/api/grantnav/updates` and the username and password can be found in [the private deploy repo](https://github.com/ThreeSixtyGiving/threesixtygiving-deploy-pillar-private).)
```
Comment (michaelwood) [inline docs/developer/docker-compose.md:31]: Prefer if you didn't say this, it's a 360 policy thing, some people do have access outside of 360 to the datastore.

**Review by michaelwood — APPROVED** [review]

**Pull request merged**